utils/validation.js

```diff
@@ -14,6 +15,15 @@ const checkString = (strVal, name) => {
   return strVal.trim();
 };
 
+const checks = (str) => {
+  if (!str) throw "You must provide a string";
+  if (typeof str !== "string") throw "Input must be a string";
+  if (str.trim().length === 0)
+    throw "Input cannot be an empty string or string with just spaces";
+};
+
+
+
```

Comment:
You can create a validation function to check the movie array parameter in create and update. The function would look like this: 
```suggestion
const checkMovieArray = (arr, name) => {
  if (!Array.isArray(arr)) throw `Error: ${name} must be an array`;

  if (arr.length === 0) throw `Error: ${name} must have at least one element`;

  for (let index in arr) {
    arr[index] = checkMovieId(arr[index]);
  }

  return arr;
}
```

Be sure to export it too